cmd/openshift-install/testdata/agent/image/validations/compact_invalid_controlPlane_replicas.txt

```diff
@@ -0,0 +1,42 @@
+! exec openshift-install agent create image --dir $WORK
+
+stderr 'ControlPlane.Replicas can only be set to 3 or 1. Found 2'
```

Comment:
@bfournie I think the error message should be like `ControlPlane.Replicas can only be set to 1 for SNO and a minimum 3 in case of HA and Compact clusters. Found 2` Thoughts?